### PR TITLE
Filter legacy

### LINKED ISSUE #1900: Exclude dart2-incompatible packages from search
We should probably start excluding all dart2-incompatible packages from searches

### PR DESCRIPTION
**Pull request by jonasfj**
Fix #1900.

It's possible we should be setting `includeLegacy: true`  when listing packages.

### CODE REVIEW

**Comment by isoos** [comment]
> It's possible we should be setting includeLegacy: true when listing packages.

I'm a bit undecided on that. Maybe let's start with true for now, and do a switch later?

**Comment by jonasfj** [comment]
Let's add a checkbox to search entry... that way there is visibility into what is happening here.

File: static/css/style.css [hunk 1]
```diff
@@ -1194,7 +1194,7 @@ pre > code {
           align-items: center;
   border-radius: 3px;
   background: #f5f5f5;
-  max-width: 550px;
+  max-width: 650px;
```
Comment (isoos) [inline static/css/style.css]: why the change?

Comment (jonasfj) [reply static/css/style.css]: because there wasn't room for the two check boxes...

Comment (isoos) [reply static/css/style.css]: SG.
